user/encryption-keys.md

```diff
@@ -37,6 +37,11 @@ secure: ".... encrypted data ...."
 
 Now you can place it in the `.travis.yml` file.
 
+> Pro Tip: You can also have travis add it automatically by running the command as follows:
```

Comment:
Thanks for this.

We avoid phrases like "Pro Tip" or "Just". And always use Travis CI not Travis.

How about running the text on onto the previous sentence like

 Now you can place it in the `.travis.yml` file. Or add it to your `.travis.yml` by running `travis encrypt SOMEVAR="secretvalue" --add` instead.